gradle/lib/dependabot/gradle/file_updater/lockfile_updater.rb

```diff
@@ -0,0 +1,62 @@
+# typed: strong
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+require "shellwords"
+
+require "dependabot/gradle/file_parser"
+require "dependabot/gradle/file_updater"
+
+module Dependabot
+  module Gradle
+    class LockfileUpdater
```

Comment:
I think that the Dependabot style is to make this a child under `FileUpdater`, not directly `Gradle`.